envoy/api/v2/discovery/BUILD

```diff
@@ -0,0 +1,155 @@
+load("//bazel:api_build_system.bzl", "api_proto_library", "api_go_proto_library", "api_go_grpc_library")
```

Comment:
We could also make this `envoy.api.v2.services` and put metrics/trace services and RLS here, WDYT? I don't have a strong preference.

Reply:
I think `discovery` is more semantically loaded than `services`, so is a better choice.